platform/landing-page/src/main/java/org/codice/ddf/landing/LandingPage.java

```diff
@@ -102,6 +109,32 @@
 
   private String foreground;
 
+  public void setSourceKeyword(ResourceBundleLocator resourceBundleLocator) {
+    try {
+      ResourceBundle resourceBundle = resourceBundleLocator.getBundle(LANDING_PAGE_BASE_NAME);
+
+      if (resourceBundle.containsKey("dataSource")) {
+        this.sourceKeyword = (String) resourceBundle.getObject("dataSource");
```

Comment:
✏️ I would recommend calling `getString()` instead.

BTW, there is no guarantee that a bundle entry is a string. It could be an array of strings or an object by design.